docker-compose.yml

```diff
@@ -0,0 +1,206 @@
+version: '2'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.3.0 #'bitnami/zookeeper:3'
+    hostname: zookeeper
+    container_name: zookeeper
+    ports:
+      - 2181:2181
```

Comment:
There are several ports opened to public internet in this docker-compose setup. Do they really need to be open to the public, or would 127.0.0.1 be enough?  (The publicly opened ports I see in this config are: 2181, 29002, 9092, 5432, 8000, 8088, 8081, 5430, 9200, 5601)